README.rst

```diff
@@ -72,7 +72,7 @@ How To Contribute
 -----------------
 
 Contributions are very welcome.
-Please read `How To Contribute <https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst>`_ for details.
+Please read `Contribution Guidelines <https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst>`_ for details.
```

Comment:
There should be a way to fix this test failure without modifying the text too. I'll have a look at it to see if we can fix the failure without changing the description.